kubernetes/samples/scripts/create-soa-domain/README.md

```diff
@@ -1,4 +1,10 @@
-# Oracle SOA on Kubernetes
+---
+title: "Oracle SOA on Kubernetes"
+date: 2019-12-05T06:46:23-05:00
+weight: 2
+description: "Deployment of SOA domains on Kubernetes cluster using Oralce WebLogic operator"
```

Comment:
on Kubernetes cluster using Oralce WebLogic operator -> on a Kubernetes cluster using the Oracle WebLogic Kubernetes Operator